docs/source/bricks.rst

```diff
@@ -338,6 +338,24 @@ Examples:
   isd = convert_to_isd(elements)
 
 
+``isd_to_elements``
+-------------------
+
+Converts outputs from initial structured data (ISD) format back to a list of ``Text`` elements.
+
+Examples:
+
+.. code:: python
+
+  from unstructured.staging.base import isd_to_elements
+
+  isd = [
+    {"text": "My Title", "type": "Title"},
+    {"text": "My Narrative", "type": "NarrativeText"}
+  ]
+  elements = isd_to_elements(isd)
+
```

Comment:
maybe you could comment on what `elements` is here
```
[ Title(text="My Title"),
   NarrativeText(text="My Narrative")]
 ```

Reply:
Added, thanks!